pallets/parachain-staking/src/weights.rs

```diff
@@ -77,241 +78,235 @@ pub trait WeightInfo {
+		(37_237_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	fn candidate_bond_more() -> Weight {
-		(59_735_000 as Weight)
+		(60_957_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(8 as Weight))
 			.saturating_add(T::DbWeight::get().writes(6 as Weight))
 	}
 	fn candidate_bond_less() -> Weight {
-		(59_421_000 as Weight)
+		(60_829_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(8 as Weight))
 			.saturating_add(T::DbWeight::get().writes(6 as Weight))
 	}
 	fn nominate(x: u32, y: u32) -> Weight {
-		(71_656_000 as Weight)
-			// Standard Error: 1_000
-			.saturating_add((1_049_000 as Weight).saturating_mul(x as Weight))
-			// Standard Error: 5_000
-			.saturating_add((947_000 as Weight).saturating_mul(y as Weight))
+		(86_279_000 as Weight)
+			// Standard Error: 3_000
+			.saturating_add((662_000 as Weight).saturating_mul(x as Weight))
+			// Standard Error: 3_000
+			.saturating_add((716_000 as Weight).saturating_mul(y as Weight))
 			.saturating_add(T::DbWeight::get().reads(9 as Weight))
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
 	fn leave_nominators(x: u32) -> Weight {
-		(36_354_000 as Weight)
+		(39_624_000 as Weight)
 			// Standard Error: 2_000
-			.saturating_add((694_000 as Weight).saturating_mul(x as Weight))
+			.saturating_add((419_000 as Weight).saturating_mul(x as Weight))
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	fn revoke_nomination() -> Weight {
-		(37_580_000 as Weight)
+		(38_159_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(7 as Weight))
 			.saturating_add(T::DbWeight::get().writes(4 as Weight))
 	}
 	fn nominator_bond_more() -> Weight {
-		(70_867_000 as Weight)
+		(71_358_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(9 as Weight))
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
 	fn nominator_bond_less() -> Weight {
-		(70_857_000 as Weight)
+		(71_183_000 as Weight)
 			.saturating_add(T::DbWeight::get().reads(9 as Weight))
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
-	// If this takes up too much block space, run again on code
-	// - #743 benchmarks post reward payout optimization were 3x lower per collator,
-	// 15x lower per nominator
 	fn active_on_initialize(x: u32, y: u32) -> Weight {
 		(0 as Weight)
-			// Standard Error: 299_000
-			.saturating_add((208_550_000 as Weight).saturating_mul(x as Weight))
-			// Standard Error: 27_000
-			.saturating_add((15_580_000 as Weight).saturating_mul(y as Weight))
-			.saturating_add(T::DbWeight::get().reads(26 as Weight))
+			// Standard Error: 778_000
+			.saturating_add((134_913_000 as Weight).saturating_mul(x as Weight))
+			// Standard Error: 7_000
+			.saturating_add((876_000 as Weight).saturating_mul(y as Weight))
+			.saturating_add(T::DbWeight::get().reads(101 as Weight))
```

Comment:
@4meta5 do you understand why we see more db reads (`26 -> 101`) and writes (`16 -> 90`) now?

Reply:
Every benchmarking run produces different results. In this run, the results used larger values for the db reads coeffcient and lower values for the added constants:
```
old_constant_added_value < new_constant_added_value
134_913_00 < 208_550_000
876_000 < 15_580_000
```

Reply:
Yes, this seems to have shifted some time -> db operations, but why?

Reply:
I have no clue. Benchmarking moves things around like this on every run. I've guessed that's why substrate often runs the bench bot more than once per PR.

A long standing question I've had is whether to prefer the first benchmarking run or subsequent runs. Because the first benchmarking run is always the worst and then it tails off after the 2nd.

Reply:
I've seen it bump the weight related numbers around, but not the db read/write count before.

I'll rerun and see what results we get...

Reply:
The second run produced very similar numbers, esp. related to `active_on_initialize`